stqry.js

```diff
@@ -115,26 +137,27 @@ window.stqry = {
       */
     get: function (keys, callback, customKey) {
       var storageKey = customKey || STORAGE_KEY
-      if (window.stqryRuntime === "ReactNative") {
-        callReactNative("storage.get", { keys: keys, storageKey: storageKey }, callback)
+      if (window.stqryRuntime === 'testing') {
```

Comment:
@PanBachynskyi just not sure about this name. Maybe we rename it to "NoRuntime"?

Reply:
@noway I've changed it